Feature/signering release (#2177)
Altinn/altinn-studio-docs: content/altinn-studio/guides/development/signing/role-based-signing/backend-manual/add-process-task.en.md

          <altinn:dataType>ref-data-as-pdf</altinn:dataType>
        </altinn:dataTypesToSign>
        <altinn:signatureDataType>signatures</altinn:signatureDataType>

>         <!-- We have made a default validator that can be enabled here. If this is disabled, custom validation of the signatures should be added. -->

Reviewer: ```suggestion
        <!-- We have made a default validator that can be enabled here. If default validation is not enabled, custom validation of the signatures should be added. -->
```

Reviewer: Suggested change because the default validator is disabled by default

  "id": "signatures",
  "allowedContentTypes": [
      "application/json"
  ],

>   "allowedContributers": [

Reviewer 2: contributers og contributors?

Author: We are working on support both, but currently contributers . :P

Author: https://github.com/Altinn/app-lib-dotnet/issues/1301

Author: Will update docs when we support the correct spelling as well.

Reviewer: We do support both: https://github.com/Altinn/app-lib-dotnet/pull/956/commits/943dd79ca25365ad37e8fe4f9f5940d2adee604c

Author: Ah nice!

Author: Updated docs.
